### DISCUSSION TITLE
Übernahme von Punkten in Redlining analog Flächen und Linien?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#148] [Q&A]
Flächen und Linien können komfortabel über "Flächen/Linien aus Selektion xxx übernehmen..." ins Redlining übernommen werden.
Das funktioniert mit punktförmigen Inhalten nur, wenn das Werkzeug "Symbol" gewählt wird. Für das einfache Darstellen von farbigen Punkten (analog der Darstellung von Flächen und Linien) ist das umständlich, weil man die Größe und die Farbe nicht variieren kann  (für jede Darstellung muss ein eigenes Symbol erstellt werden).
Darf ich mir die Übernahme von Punkten (analog Flächen/Linien) wünschen?

![Grundstück-Selektion](https://github.com/gis-eni/webgis-forum/assets/100088983/cbb6877c-46e2-4434-888c-f76767c74a59)

**Comment by gis-eni** [comment]
Im nächsten Build mit dabei:
![image](https://github.com/gis-eni/webgis-forum/assets/62870568/ee7cb47b-221e-43da-9d19-c853559389df)

**Reply by faulgurke** [reply]
Ist das Build der 5_24_1001 oder der nächste?

**Reply by gis-eni** [reply]
klar

**Reply by ue-black** [reply]
Das funktioniert bei uns leider noch nicht. Nach dem Drücken des Buttons "Aus Selektion übernehmen" passiert nichts mehr. Hast du eine Ahnung woran das liegen könnte?

https://github.com/gis-eni/webgis-forum/assets/127403099/4c2addc5-9b87-4a1f-8275-13a9dfa83029

**Reply by gis-eni** [reply]
Wen "nichts" passiert kann ich leider relativ wenig dazu sagen. Eventuell sieht man einen Javascript Error in der Konsole?

**Reply by ue-black** [reply]
Stimmt, das ist schwierig :-)
Hier die Antworten von der Konsole. Es scheint als ob bei den Punkten keine Geometrien mitgeliefert werden:
Fehlermeldung Console:
![console](https://github.com/gis-eni/webgis-forum/assets/127403099/45db37dc-3468-496c-be8c-3c3df93b4f53)

Toolevent Punkte:
![punkte](https://github.com/gis-eni/webgis-forum/assets/127403099/7e9e0b32-1fa0-4042-86e8-1c8a99de80ec)

Toolevent Polygon:
![polygon](https://github.com/gis-eni/webgis-forum/assets/127403099/f84827e7-c692-42fe-b562-685e7c37b395)

**Reply by gis-eni** [reply]
Hm. Kann ich leider nicht nachvollziehen. Hab es gerade noch einmal bei mir probiert. Warum die Geometrie bei dir "null" ist?! Keine Ahnung. Was für eine Art von Kartendienst ist da dahinter?

**Reply by faulgurke** [reply]
Der Kartendienst ist gView, Version 5.24.201. Der Geometrietyp ist Multipoint.
Da hat's dann bei uns geklingelt. Ein Test mit Geometrietyp "Point" hat dann funktioniert.
Jetzt kommt natürlich die nächste Frage: kann man die Funktionalität "Übernahme von Punkten in Redlining" auch für Multipoint-features ermöglichen?

**Reply by gis-eni** [reply]
Das war auch mein Verdacht :) 
Multipoints funktionieren natürlich nicht. Ein Punkt ist immer EIN Punkt. Eine Multipoint Geometrie gibt es im Redlining nicht. Die Frage ist, wie sollten wir hier mit Multipoints im Redlining umgehen?
* nur den ersten übernehmen?
* Für jeden Punkt im Multipoint einen Punkt anlegen. Dann man halt mehr Punkte als Objekte.

Und habt ihr vielleicht einen Dienst bei euch mit Multipoints, mit dem ich das Testen kann? Wir haben in unseren Datenbank nirgends sowas herumliegen

**Reply by faulgurke** [reply]
Zur Frage, wie man mit Multipoints im Redlining umgehen soll:
Für jeden Punkt im Multipoint einen Punkt anlegen wäre für uns die beste Variante - da geht es ja primär um die Darstellung von Objekten (also wo befindet sich der Knödel).
Zum Dienst:
Den schicke ich dir per email.
Bemerkung zu Multipoints: "Richtige" Multipoints sind bei uns die Ausnahme. Man hat der Einfachheit halber den Prozess der Datenübernahme seinerzeit auf Multipoint eingestellt.

**Reply by gis-eni** [reply]
Ok, ich konnte es so umsetzen, dass es bei MultiPoints, die nur einen Punkt enthalten funktioniert. Für "echte" Multipoints funktioniert das auch in Zukunft nicht. Aber kommen eh nur sehr selten vor.

**Reply by faulgurke** [reply]
Sollte die Version 5.24.1101 dieses feature schon unterstützen? 
Ich schaffe es noch immer nicht selektierte Punkte ins Redlining (Punkte) zu übernehmen. 
Übernahme in Symbole funktioniert.

**Reply by faulgurke** [reply]
Entschuldige bitte die nochmalige Nachfrage, aber hat das bei dir mit dem von mir bereitgestellten Service funktioniert?
Bei mir klappt nämlich die Übernahme noch immer nicht (Version 5.24.1101).

**Reply by gis-eni** [reply]
Ja, hat bei mir geklappt

**Reply by gis-eni** [reply]
Meine Antwort stammt aber vom 20.3 => Kalenderwoche 12. Der Build ist aber noch 11xx. Vielleicht liegt es daran

**Reply by faulgurke** [reply]
Ach so, dann ist dieses Feature noch nicht im aktuellen Build vorhanden. Dann stelle ich meine (vergeblichen) Versuche ein ;-)
